docs/docs/users/set-password-oauth-users.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Set Password for OAuth Users
+intro: Learn how users who signed up via Google OAuth can set a password to access the API with email/password authentication.
```

Comment:
I am wondering ... from security standpoint, do we really want to encourage a password based approach, when users can sign up/login using a password-less approach and also access the API using API keys/tokens, as documented in https://devopness.com/docs/api-tokens/ ?

Would there be any benefits of setting up passwords for existing password-less accounts, instead of using API Tokens?